Userland/Services/Taskbar/TaskbarWindow.cpp

```diff
@@ -82,15 +82,15 @@ ErrorOr<void> TaskbarWidget::create_context_menu()
 {
     m_context_menu = GUI::Menu::construct();
 
-    auto show_desktop_action = GUI::Action::create("&Show Desktop", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/desktop.png"sv)), [](auto&) {
+    auto show_desktop_action = GUI::Action::create("Show &Desktop", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/desktop.png"sv)), [](auto&) {
         GUI::ConnectionToWindowManagerServer::the().async_toggle_show_desktop();
     });
 
     auto open_settings_action = GUI::Action::create("&Settings", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/settings.png"sv)), [this](auto&) {
         GUI::Process::spawn_or_show_error(window(), "/bin/Settings"sv);
     });
 
-    auto open_system_monitor_action = GUI::Action::create("System &Monitor", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/app-system-monitor.png"sv)), [this](auto&) {
+    auto open_system_monitor_action = GUI::Action::create("&System Monitor", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/app-system-monitor.png"sv)), [this](auto&) {
```

Comment:
Since this is a change in code you introduced in this PR, you should not make a separate commit but instead squash the changes into the previous commit.